draft-ietf-rats-uccs.md

```diff
@@ -262,6 +266,12 @@ endorsement of the Secure Channel.  (Note that EAT might add a nested UCCS
 Claim, and this statement does not apply to UCCS nested into UCCS, only to
 fully formed CWTs)
 
+### Delegated Attestation
+
+Another use case is that of a sub-Attester that has no signing keys (for example, to keep the implementation complexity to a minimum) and has a secure channel, such as a local IPC, to interact with a lead Attester (see Composite Device, {{Section 3.3 of -rats}}).
```

Comment:
SPDM connected components over a system bus such as MCTP is another concrete example

Reply:
Arm CCA attestation in "direct" mode another.